### DISCUSSION TITLE
PHP Error on Dashboard

### DISCUSSION THREAD

**Discussion by KW4G** [magicbug/Cloudlog#3197] [Q&A]
I am getting the following error message on my dashboard. I have CloudLog installed on a commercial server. I am not a coder, php or otherwise, so please be kind.
![Screenshot 2024-09-27 at 10-29-13 Dashboard - Cloudlog](https://github.com/user-attachments/assets/c967f83e-6364-44f2-a91b-1896eb8f36b1)

**Comment by magicbug** [comment]
This is likely caused by PHP version being outside of the recommended version https://github.com/magicbug/Cloudlog see readme for more information.

If it cannot be resolved you could turn off the Upcoming DXPeditions card in account settings

**Reply by KW4G** [reply]
I changed my PHP to version 7.4 and the error seems to have gone away. Thanks.
